app/forms/mappings/LocalDateFormatter.scala

```diff
@@ -33,15 +33,15 @@ private[mappings] class LocalDateFormatter(
   val log: LoggerLike = Logger(this.getClass)
   val currentDate: LocalDate = LocalDateTime.now().toLocalDate
 
-  private def getEndDay(month: Int, year: Int, date: LocalDate): Int = {
-    if (month == currentDate.getMonthValue && year == currentDate.getYear) {
-      log.info("entered month: " + month + "; entered year: " + year)
-      log.info("current month: " + currentDate.getMonthValue + "; currentyear: " + currentDate.getYear)
-      currentDate.getDayOfMonth
-    } else {
-      date.lengthOfMonth()
-    }
-  }
+//  private def getEndDay(month: Int, year: Int, date: LocalDate): Int = {
```

Comment:
It can be removed if this method is not being used

Reply:
Is it being commented as we need to come back to it? If so a TODO might be better than deleting it

Reply:
yepp that's better , thanks @AdamCoops